tokio-console/src/state/async_ops.rs

```diff
@@ -245,25 +245,29 @@ impl AsyncOp {
     }
 
     pub(crate) fn total(&self, since: SystemTime) -> Duration {
-        self.stats
-            .total
-            .unwrap_or_else(|| since.duration_since(self.stats.created_at).unwrap())
+        self.stats.total.unwrap_or_else(|| {
+            since
+                .duration_since(self.stats.created_at)
+                .unwrap_or_default()
+        })
     }
 
     pub(crate) fn busy(&self, since: SystemTime) -> Duration {
         if let (Some(last_poll_started), None) =
             (self.stats.last_poll_started, self.stats.last_poll_ended)
         {
-            let current_time_in_poll = since.duration_since(last_poll_started).unwrap();
+            let current_time_in_poll = since.duration_since(last_poll_started).unwrap_or_default();
             return self.stats.busy + current_time_in_poll;
         }
         self.stats.busy
     }
 
     pub(crate) fn idle(&self, since: SystemTime) -> Duration {
-        self.stats
-            .idle
-            .unwrap_or_else(|| self.total(since) - self.busy(since))
+        self.stats.idle.unwrap_or_else(|| {
+            self.total(since)
+                .checked_sub(self.busy(since))
+                .unwrap_or_default()
+        })
```

Comment:
nit, take it or leave it: again, we could move the `unwrap_or_default` out using an `or_else` instead, which i think maybe reads a little nicer?
```suggestion
        self.stats
            .idle
            .or_else(|| self.total(since).checked_sub(self.busy(since)))
            .unwrap_or_default()
```